packages/url-state-provider/src/index.ts

```diff
@@ -122,17 +138,17 @@ function currentState(stateID) {
  * @param {JSON} state
  * @returns new url string
  */
-function updateState(stateID, state, options = {}) {
+function updateState(stateID: string, state: object, options: UpdateStateOptions = {}) {
   // get current state from URL
-  const newState = URLToState()
+  const newState: { [key: string]: any } = URLToState()
```

Comment:
unknown

Reply:
if i do that there is the error "spread can only be created from object types.